docker-for-windows/index.md

```diff
@@ -512,44 +512,41 @@ For more information on Windows containers, refer to this documentation:
   application with Windows containers. The MusicStore is a standard .NET application and,
   [forked here to use containers](https://github.com/friism/MusicStore), is a good example of a multi-container application.
 
-- To understand how to connect to Windows containers from the local host, see:
+- To understand how to connect to Windows containers from the local host, see
   [Limitations of Windows containers for `localhost` and published ports](troubleshoot.md#limitations-of-windows-containers-for-localhost-and-published-ports)
 
 > Settings dialog changes with Windows containers
 >
 > When you switch to Windows containers, the Settings dialog only shows those tabs that are active and apply to your Windows containers:
 >
+
   * [General](#general)
   * [Proxies](#proxies)
-  * [Docker daemon](#docker-daemon)
-  * [Diagnose and Feedback](#diagnose-and-feedback)
+  * [Daemon](#docker-daemon)
   * [Reset](#reset)
->
+
 If you set proxies or daemon configuration in Windows containers mode, these
 apply only on Windows containers. If you switch back to Linux containers,
 proxies and daemon configurations return to what you had set for Linux
 containers. Your Windows container settings are retained and become available
 again when you switch back.
->
-
-## Adding TLS certificates
-
-To your Docker daemon, you can add trusted **Certificate Authorities (CAs)**, to verify registry server
-certificates, and **client certificates**, to authenticate to registries.
-
-See [How do I add custom CA certificates?](faqs.md#how-do-i-add-custom-ca-certificates)
-and [How do I add client certificates?](faqs.md#how-do-i-add-client-certificates)
-in the FAQs.
 
 ## Docker Hub
 
-Select **Sign in /Create Docker ID** from the Docker Desktop for Windows menu to access your [Docker Hub](https://hub.docker.com/){: target="_blank" clas="_" } account. Once logged in, you can access your Docker Hub repositories directly from the Docker Desktop for Windows menu.
+Select **Sign in /Create Docker ID** from the Docker Desktop menu to access your [Docker Hub](https://hub.docker.com/){: target="_blank" class="_" } account. Once logged in, you can access your Docker Hub repositories directly from the Docker Desktop menu.
 
-See these [Docker Hub topics](/docker-hub/index.md){: target="_blank" class="_" } to learn more:
+For more information, refer to the following [Docker Hub topics](/docker-hub/index.md){: target="_blank" class="_" }:
 
 * [Organizations and Teams in Docker Hub](/docker-hub/orgs.md){: target="_blank" class="_" }
 * [Builds and Images](/docker-cloud/builds/index.md){: target="_blank" class="_" }
 
+## Adding TLS certificates
+
+You can add trusted **Certificate Authorities (CAs)** to your Docker daemon to verify registry server
```

Comment:
This is the second occurrence of certificate authorities, so I would suggest using "CA here, unless this is a specific user action.

Reply:
The first occurrence is a link "How do I add custom CA certificates?" in the "Insecure Registries" section. Therefore, I'm tempted to keep it as is.